src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.model;
+
+import java.util.Objects;
+
+public class Book {
+    private Long id;
+    private String title;
+    private int price;
```

Comment:
The `price` field should be of type `BigDecimal` instead of `int` to handle monetary values accurately.